packages/discovery-provider/plugins/pedalboard/apps/mri/src/date.ts

```diff
@@ -0,0 +1,19 @@
+const padToTwoDigits = (num: number) => {
```

Comment:
lint is weird on this file, maybe my bad

Reply:
ah yes ty